linkml_model/model/schema/meta.yaml

```diff
@@ -2410,3 +2534,33 @@ enums:
         meaning: skos:broaderMatch
       NARROW_SYNONYM:
         meaning: skos:narrowerMatch
+
+  # I'm not really sure if this is valid/correct, but I wanted to give a quick and dirty example. This is based on: https://build.fhir.org/valueset-example-intensional.json.html - joeflack4 2022/05/27
+  serum_cholesterol_enum:
+    # `description`: I didn't see this as a valid `slot` for `enum_expression`. Should this be here?
+    description: >
+      This is an example value set that includes all the LOINC codes for serum/plasma cholesterol from v2.36.
+    # `is_a`: This too. is this / should this be here?
+    is_a: enum_expression
```

Comment:
`is_a`: is / can / should this be here?

Reply:
Never mind. I got an error: `ValueError:  Unknown argument: is_a = 'enum_expression'`

So I commented this out. I feel like it would be nice to have a field on enums though to show its superclass; that is, to know whether it is an intensionally or extensionally defined enum just by looking at this 1 field.